plugins/plugin-grid/src/lib/cmds/table.ts

```diff
@@ -653,7 +653,5 @@ export default async (commandTree: CommandRegistrar) => {
     height: windowDefaults.height
   }
 
-  const cmd = commandTree.listen(`/wsk/table`, tableIt('table'), opts)
```

Comment:
This will remove the wsk table command listener. I think removing `const cmd =` would be enough. Thanks!

Reply:
Other changes look good!

Reply:
It's solved in the next commit.

Reply:
Squash ready